### PR TITLE
Solution for part 1

### PR DESCRIPTION
**Pull request by Paul-Melinte**
Had to copy to this repo the solution, it took longer that the timestamps show 😃

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function fizzBuzz(n: number) {
+    for(let i = 1;i <= n;i++){
+        const strings = []; // Using a array of strings to save the different parts of the answer
+        //Decided to do this because we can't change a const and I didn't want to keep track of which string we have to print at the end
```
Comment (cipacda) [inline index.ts]: You will want to leave a space between // and the comment

Generally it's best to put the comments on the line before the commented on line

File: index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        
+        if(i % 3 == 0)
+            strings.push("Fizz");
+
+        if(i % 5 == 0)
+            strings.push("Buzz");
+
+        if(strings.length == 0)
+            console.log(i);
+        else console.log(strings.join(""));
```
Comment (cipacda) [inline index.ts]: Some inconsistency here - console.log inside if is on new line, and the one inside else it's on the same line

File: index.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+function fizzBuzz(n: number) {
+    for(let i = 1;i <= n;i++){
```
Comment (cipacda) [inline index.ts]: you will want spaces after semi-colons

**Comment by Paul-Melinte** [comment]
Wrong branch for the last commit, I'm not sure how to roll back on GitHub

**Review by cipacda — APPROVED** [review]
Great, does the job - although probably on the too smart side on how it's implemented and not the easiest to follow (might be easier to just have a sequence of if and elseif) for all scenarios

**Pull request merged**